metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java

```diff
@@ -4062,4 +4062,26 @@ public void shouldUriEncodePathSegmentWithoutSafeChars() {
         );
     }
 
+    @Test
+    public void shouldTransformStringToBase64() {
+        MetafixTestHelpers.assertFix(streamReceiver, Arrays.asList(
+                "to_base64('data.title')"
+            ),
+            i -> {
+                i.startRecord("1");
+                i.startEntity("data");
+                i.literal("title", "this-is-a-test");
+                i.endEntity();
+                i.endRecord();
+            },
+            o -> {
+                o.get().startRecord("1");
+                o.get().startEntity("data");
+                o.get().literal("title", "blabka=");
```

Comment:
> do you have an idea why the build failes?

In 59ce50c you changed the Base64 value to an incorrect one. I was already wondering what that was about...